MoviesTVSentiments/app/src/main/java/com/google/moviestvsentiments/service/account/AccountDao.java

```diff
@@ -18,9 +18,10 @@
      * Adds the given account name into the accounts table. If the name already exists,
      * it is ignored. The account record is created with is_current set to false.
      * @param name The name of the account to add.
+     * @param isPending Whether the account needs to be synced with the server or not.
      */
-    @Query("INSERT OR IGNORE INTO accounts_table (account_name) VALUES (:name)")
-    public abstract void addAccount(String name);
+    @Query("INSERT OR IGNORE INTO accounts_table (account_name, is_pending) VALUES (:name, :isPending)")
+    public abstract void addAccount(String name, boolean isPending);
```

Comment:
Looking around, I wonder if this is the only addAccount method we need? 

It is also a bit confusing to have the below addAccounts(List<Account> accounts), where we can add more than one account and potentially some pending and others not?

Reply:
Ignoring the confusing part, I can hardly come up with a case where the app may need to add multiple accounts at one time?

Reply:
Are you suggesting to remove the addAccounts(List accounts) method?

Reply:
The only case I can think of is when the app pulls the list of accounts from the server. If it doesn't already have all those accounts locally, it will need to add each of them.

Reply:
Yes, that is the only case I found that is currently calling addAccounts.

I don't have a super strong preference on whether to remove this method or not. It just seems to me that it is not much used, and caller can call AddAccount multiple times with the same result. But I think it is ok to keep since this is just a Dao interface.